website_menu_icon/static/src/xml/website.contentMenu.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <div t-name="website.contentMenu.entry.dialog.icon" t-extend="wysiwyg.widgets.link">
+        <t t-jquery="form" t-operation="append">
+            <div class="form-group row">
+                <label
+                    class="col-form-label col-md-3"
+                    for="o_icon_dialog_label_input"
+                >Icon / Image</label>
+                <div class="col-md-9 input-group ml-auto">
+                    <input
+                        type="text"
+                        name="image"
+                        id="o_icon_dialog_label_input_value"
+                        class="form-control o_we_search"
+                        placeholder="Search icon or image"
+                        readonly="1"
+                        t-att-value="widget.data.image"
+                    />
+                    <div class="input-group-append">
+                        <div class="input-group-text o_we_search_icon">
+                            <i
+                                class="fa fa-search"
+                                title="Search"
+                                role="img"
+                                aria-label="Search"
+                            />
+                        </div>
+                        <div class="input-group-text o_we_delete_icon">
+                            <i
+                                class="fa fa-remove"
+                                title="Remove"
+                                role="img"
+                                aria-label="Remove"
+                            />
+                        </div>
```

Comment:
These are not text addons, they are buttons. Also, replace the :x: for a :wastebasket: which is more analogous to the rest of odoo UI:

```suggestion
                        <button type="button" class="btn btn-primary o_we_search_icon">
                            <i
                                class="fa fa-search"
                                title="Search"
                                role="img"
                                aria-label="Search"
                            />
                        </button>
                        <button type="button" class="btn btn-secondary o_we_delete_icon">
                            <i
                                class="fa fa-trash"
                                title="Remove"
                                role="img"
                                aria-label="Remove"
                            />
                        </button>
```